WordPress/src/main/AndroidManifest.xml

```diff
@@ -1148,11 +1148,6 @@
                 tools:node="remove" />
         </provider>
 
-        <!-- Scanner -->
```

Comment:
This `meta-data` is unnecessary. The QR code scanner comes in two forms:

- Bundled scanner
- Google Play Store scanner

We use the former and [only the latter requires this](https://developers.google.com/ml-kit/vision/barcode-scanning/android#before-you-begin).